<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCopySettingsAndCachesComponent</class>
 <widget class="QFrame" name="CCopySettingsAndCachesComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="vl_CopySettingsAndCache" stretch="0,0">
   <item>
    <widget class="QGroupBox" name="gb_OtherSwiftVersions">
     <property name="title">
      <string>Select the version to copy from</string>
     </property>
     <layout class="QVBoxLayout" name="vl_OtherSwiftVersions">
      <item>
       <widget class="swift::gui::components::COtherSwiftVersionsComponent" name="comp_OtherSwiftVersions">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>125</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_WhatToCopy">
     <property name="title">
      <string>What to copy?</string>
     </property>
     <layout class="QGridLayout" name="gl_WhatToCopy">
      <item row="0" column="2">
       <widget class="QGroupBox" name="gb_Model">
        <property name="title">
         <string>Models</string>
        </property>
        <layout class="QVBoxLayout" name="vl_Models">
         <item>
          <widget class="QCheckBox" name="cb_SettingsModel">
           <property name="text">
            <string>Model</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_SettingsConsolidation">
           <property name="text">
            <string>Consolidation</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="gb_Audio">
        <property name="title">
         <string>Audio</string>
        </property>
        <layout class="QVBoxLayout" name="vl_Audio">
         <item>
          <widget class="QCheckBox" name="cb_SettingsAudio">
           <property name="text">
            <string>Audio1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_SettingsAudioInputDevice">
           <property name="text">
            <string>Audio2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_SettingsAudioOutputDevice">
           <property name="text">
            <string>Audio3</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QFrame" name="fr_Buttons">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="le_Status">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_DeselectAll">
           <property name="text">
            <string>deselect all</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_SelectAll">
           <property name="text">
            <string>select all</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_Copy">
           <property name="text">
            <string>copy</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QFrame" name="fr_Middle">
        <layout class="QVBoxLayout" name="vl_Middle">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="gb_Simulator">
           <property name="title">
            <string>Simulator</string>
           </property>
           <layout class="QVBoxLayout" name="vl_Simulator">
            <item>
             <widget class="QCheckBox" name="cb_SettingsEnabledSimulators">
              <property name="text">
               <string>Enabled sims</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cb_SettingsSimulatorFSX">
              <property name="text">
               <string>FSX</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cb_SettingsSimulatorP3D">
              <property name="text">
               <string>P3D</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cb_SettingsSimulatorXPlane">
              <property name="text">
               <string>XPlane</string>
              </property>
             </widget>
            </item>
 		    <item>
			 <widget class="QCheckBox" name="cb_SettingsSimulatorFG">
			  <property name="text">
			   <string>FG</string>
			  </property>
			 </widget>
		   </item>
           <item>
  			 <widget class="QCheckBox" name="cb_SettingsSimulatorMSFS2020">
			  <property name="text">
				<string>MSFS2020</string>
			  </property>
			 </widget>
			</item>
			<item>
			 <widget class="QCheckBox" name="cb_SettingsSimulatorMSFS2024">
			  <property name="text">
				<string>MSFS2024</string>
			  </property>
			 </widget>
			</item>
		   </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QFrame" name="fr_Right">
        <layout class="QVBoxLayout" name="vl_Right">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="gb_Misc">
           <property name="title">
            <string>Misc.</string>
           </property>
           <layout class="QVBoxLayout" name="vl_Misc">
            <item>
             <widget class="QCheckBox" name="cb_SettingsActionHotkeys">
              <property name="text">
               <string>Keys</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cb_SettingsDirectories">
              <property name="text">
               <string>Directories</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cb_SettingsTextMessages">
              <property name="text">
               <string>Text Messages</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="gb_Network">
        <property name="title">
         <string>Network</string>
        </property>
        <layout class="QVBoxLayout" name="vl_Network">
         <item>
          <widget class="QCheckBox" name="cb_SettingsNetworkTrafficServers">
           <property name="text">
            <string>Network1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_CacheLastNetworkServer">
           <property name="text">
            <string>Network2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_CacheLastVatsimServer">
           <property name="text">
            <string>Network3</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="fr_Left">
        <layout class="QVBoxLayout" name="vl_left">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="gb_Ui">
           <property name="title">
            <string>UI</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QCheckBox" name="cb_SettingsGuiGeneral">
              <property name="text">
               <string>UI1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cb_SettingsDockWidget">
              <property name="text">
               <string>UI2</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cb_SettingsViewUpdate">
              <property name="text">
               <string>UI3</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::components::COtherSwiftVersionsComponent</class>
   <extends>QFrame</extends>
   <header>gui/components/otherswiftversionscomponent.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cb_SettingsAudio</tabstop>
  <tabstop>cb_SettingsAudioOutputDevice</tabstop>
  <tabstop>cb_SettingsAudioInputDevice</tabstop>
  <tabstop>cb_SettingsGuiGeneral</tabstop>
  <tabstop>cb_SettingsDockWidget</tabstop>
  <tabstop>cb_SettingsViewUpdate</tabstop>
  <tabstop>cb_SettingsNetworkTrafficServers</tabstop>
  <tabstop>cb_CacheLastNetworkServer</tabstop>
  <tabstop>cb_CacheLastVatsimServer</tabstop>
  <tabstop>cb_SettingsEnabledSimulators</tabstop>
  <tabstop>cb_SettingsSimulatorP3D</tabstop>
  <tabstop>cb_SettingsSimulatorFSX</tabstop>
  <tabstop>cb_SettingsSimulatorXPlane</tabstop>
  <tabstop>cb_SettingsSimulatorFG</tabstop>
  <tabstop>cb_SettingsSimulatorMSFS2020</tabstop>
  <tabstop>cb_SettingsSimulatorMSFS2020</tabstop>
  <tabstop>cb_SettingsModel</tabstop>
  <tabstop>cb_SettingsConsolidation</tabstop>
  <tabstop>cb_SettingsActionHotkeys</tabstop>
  <tabstop>cb_SettingsDirectories</tabstop>
  <tabstop>cb_SettingsTextMessages</tabstop>
  <tabstop>le_Status</tabstop>
  <tabstop>pb_DeselectAll</tabstop>
  <tabstop>pb_SelectAll</tabstop>
  <tabstop>pb_Copy</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
